### ISSUE TITLE
Cannot run prokka-genbank_to_fasta_db due t missing Bio/SeqIO depedency in Homebrew package

### ISSUE THREAD

**Issue by flass** [tseemann/prokka#403]
Hi,

Running the command `prokka-genbank_to_fasta_db` gives me the following error: 
```
Can't locate Bio/SeqIO.pm in @INC (you may need to install the Bio::SeqIO module) (@INC contains: /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/site_perl/5.30.0 /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0/x86_64-linux-thread-multi /home/linuxbrew/.linuxbrew/Cellar/perl/5.30.0/lib/perl5/5.30.0 /home/linuxbrew/.linuxbrew/lib/perl5/site_perl/5.30.0) at /home/linuxbrew/.linuxbrew/bin/prokka-genbank_to_fasta_db line 4.
BEGIN failed--compilation aborted at /home/linuxbrew/.linuxbrew/bin/prokka-genbank_to_fasta_db line 4.
```
this is with a version of Prokka I just installed from Homebrew:
```
$ brew info prokka
brewsci/bio/prokka: stable 1.13 (bottled)
Rapid annotation of prokaryotic genomes
https://github.com/tseemann/prokka
/home/linuxbrew/.linuxbrew/Cellar/prokka/1.13 (103 files, 522MB) *
  Poured from bottle on 2019-08-08 at 10:57:39
From: https://github.com/brewsci/homebrew-bio/blob/master/Formula/prokka.rb
==> Dependencies
Required: aragorn ✔, barrnap ✔, bioperl ✔, blast ✔, hmmer ✔, infernal ✔, minced ✔, parallel ✔, prodigal ✔, tbl2asn ✔, less ✔, perl ✔
```
other useful context information about my distribution:
```
$ uname -a
Linux pantagrueltestb 4.15.0-46-generic #49-Ubuntu SMP Wed Feb 6 09:33:07 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux

$ brew -v
Homebrew 2.1.9
Homebrew/linuxbrew-core (git revision d36fc; last commit 2019-08-07)

$ apt list *bioperl*
Listing... Done
bioperl/bionic 1.7.2-2 all
bioperl-run/bionic 1.7.1-3 all
```
Could you please help me see what's wrong?

Also/otherwise I would be happy with a suggestion of how to install Prokka in a stable manner for integration in my pipeline [Pantagruel](https://github.com/flass/pantagruel), for which we are currently building a dockerfile for stable distribution (noting that we would rather not [@pveber cannot?] use the docker version of Prokka within Pantagruel's own docker image).

Many thanks,

Florent

**Comment by tseemann** [comment]
@flass you are right - i had not patched all the other perl scripts to point to the bioperl brew package. i am preparing a new 1.13.7 brew version of prokka.

In the mean time, you can run `/usr/bin/perl  /home/linuxbrew/.linuxbrew/bin/prokka-genbank_to_fasta_db` and it might work if you have the `apt-get install perl-bioperl` installed. This forces it to use system perl.

**Comment by tseemann** [comment]
You inspired me to fix this and release prokka 1.14.0
https://github.com/tseemann/prokka/releases/tag/v1.14.0

I am building a new brew package which should fix your problem
https://github.com/brewsci/homebrew-bio/pull/729

**Comment by flass** [comment]
Thanks for the tip! Indeed summoning the system's main perl solves this issue.
Glad (and honoured!) I inspired a prokka release! Looking forward to see it packaged in Homebrew.
Cheers,
Florent
